src/components/AnonymousChat/BaseChatItem.tsx

```diff
@@ -108,6 +109,7 @@ const BaseChatItem = ({item, index, type}: BaseChatItemComponentProps) => {
       time={calculateTime(item?.updatedAt, true)}
       username={handleUserName(item, selectedChannel)}
       type={BaseChatItemTypeProps.ANON_CHAT}
+      chatItem={item}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, ensure that the `chatItem` prop is properly validated and handled in the `BaseChatItem` component.
```diff
+ chatItem={item}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->